### DISCUSSION TITLE
Option to change Vomit to something else

### DISCUSSION THREAD

**Discussion by kinostl** [OpenRCT2/OpenRCT2#19636] [Feature requests]
It can be off-putting when playing the game to see vomit all over the place. The sprite is a little unsettling, as is the vomit status. I like the mechanical implications of it though, as its the entire basis of the nausea rating. Maybe the option could toggle it so that the status is "Nauseous" and then produce a litter sprite that looks like a paper bag?

**Comment by ZehMatt** [comment]
What exactly is unsettling about this? With enough staff you should barely see any

**Reply by kinostl** [reply]
I'm new to the game and still learning those aspects, and I've run into several situations where the handymen get lost and an area turns into a pool of puke.

**Reply by ZehMatt** [reply]
There is already a penalty for having litter/puke in your park, you should also get notifications that your Guests are not happy about the state of your park. I don't see the justification for such option.

**Comment by ocalhoun6** [comment]
Sounds like a job for a plugin, maybe? All it would really need is a hook so that it can replace puke sprites with a different sprite provided by whoever made the plugin. Or possibly just replace it with one of the other litter sprites already in the game. Some of the litter sprites could be close enough to 'paper bag' to qualify.

**Comment by ZeeMaji** [comment]
I think this could be accomplished with an asset pack in the future once graphical asset packs are implemented.
That would let you just simply change the vomit sprites to something else entirely on the clientside so it would work in multiplayer as well.

**Reply by Broxzier** [reply]
This seems like the most sane way to add this ability. Something as minor as this doesn't warrant its own option, hidden or not.

**Reply by kinostl** [reply]
That sounds good to me.
